src/project.cc

```diff
@@ -478,11 +478,10 @@ std::vector<Project::Entry> LoadFromDirectoryListing(
     }
   }
 
-
   GetFilesInFolder(
       config->project_dir, true /*recursive*/, true /*add_folder_to_path*/,
       [&folder_args, &files](const std::string& path) {
-        if (SourceFileLanguage(path) != LanguageId::Unknown) {
+        if (SourceFileLanguage(path) != LanguageId::Unknown && !IsDirectory(path)) {
```

Comment:
So GetFilesInFolder is returning directories? We should probably make it just not return directories.

Reply:
The name (GetFilesInFolder) also implies it only returns files, so if we want it to continue returning directories we should rename it, ie, GetFilesAndDirectoriesInFolder

Reply:
What should it return?

Reply:
OK i am going to change the name and commit tomorrow.